src/main/java/com/gamebros/purepazaak/ui/CardGrid.java

```diff
@@ -0,0 +1,138 @@
+package com.gamebros.purepazaak.ui;
+
+import java.util.ArrayList;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+import org.newdawn.slick.Graphics;
+import org.newdawn.slick.SlickException;
+
+import com.gamebros.purepazaak.entity.Card;
+import com.gamebros.purepazaak.view.CardView;
+
+public class CardGrid {
+  private static int horizontalSpace = 10;
+
+  private static int verticalSpace = 10;
+
+  protected int x;
+
+  protected int y;
+
+  protected int width;
+
+  protected int height;
+
+  protected ArrayList<CardView> cards = new ArrayList<CardView>();
+
+  public CardGrid(int x, int y, int width, int height) {
```

Comment:
Take a VO here instead.  How about `Grid`?  It would also be used internally for children views.